install.sh

```diff
@@ -349,8 +349,7 @@ install_epel_amzn () {
 
 # Installs EPEL for RHEL/CentOS/Alma/Rocky
 install_epel () {
-  os=$1
-  ver=$2
+  ver=$1
```

Comment:
Looks like there might be `os` references down below here - L#367 for example.

Reply:
Oh you are right I didn't see that one, I have to effectively pass it down then

Reply:
Resolved